Chinese/Yifan/modernCity.md

```diff
@@ -2,7 +2,4 @@
 至尊宝：我勒个去》》》》这是神马情况。。。
 （他走到一个卖祖传贴膜的哥们前。。。）
 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.